### PR TITLE
Gluschuk lab 3, 4, 5, 6

### PR DESCRIPTION
**Pull request by andxzrey**
Added 3-6 labs. Possible warnings were corrected when I defend this labs at the university.

### CODE REVIEW

File: labs/andrey_gluschuk/lab_6/task_3/task_3.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+"""
+     Глущук Андрій 141
+     Лабораторна робота №6
+     Завдання 3
+ 5. is_prime(n: int) -> bool: n (n >= 1) є простим числом
+"""
+
+
+def is_prime(n):
```
Comment (sen-den) [inline labs/andrey_gluschuk/lab_6/task_3/task_3.py:9]: why not use typing as-is from the task?
`def is_prime(n: int) -> bool:`

File: labs/andrey_gluschuk/lab_6/task_3/task_3.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    """
+    Counts the number of digits and the sum of digits in a number.
+    Outputs a tuple of the number of digits and sum of digits.
```
Comment (sen-den) [inline labs/andrey_gluschuk/lab_6/task_3/task_3.py:12]: _Outputs a tuple_ - are you sure? 
it returns ` (count < 2)`

File: labs/andrey_gluschuk/lab_6/task_website/website.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+app = Flask(__name__)
+
+@app.route("/")
+def hello_world():
+    return "<p>Hello, World!</p>"
+
+@app.route('/about')
+def about():
+    return 'The about page'
```
Comment (sen-den) [inline labs/andrey_gluschuk/lab_6/task_website/website.py:11]: Newline is required here

**Review by sen-den — APPROVED** [review]

File: labs/andrey_gluschuk/lab_6/task_3/task_3.py (hunk 2 above)
Comment (andxzrey) [reply labs/andrey_gluschuk/lab_6/task_3/task_3.py:12]: oops, I forgot to change the docstring copied from another task, sorry

File: labs/andrey_gluschuk/lab_6/task_3/task_3.py (hunk 1 above)
Comment (andxzrey) [reply labs/andrey_gluschuk/lab_6/task_3/task_3.py:9]: I am a previous lesson already understood how to use it. I just did the task before that.

**Review by Lexus-Mihael — APPROVED** [review]
ok

**Pull request merged**